addons/mail/views/assets.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <template id="_assets_primary_variables" inherit_id="web._assets_primary_variables">
```

Comment:
We'll see runbot but I'm guessing it will be red due to the old asset being removed here.

Just the new one should be introduced, no need to merge css or anything.

Reply:
ok, didn't understand that, I'm on it